### ISSUE TITLE
The voice search icon is not displaying for some reason

### ISSUE THREAD

**Issue by chrisjeon** [MiguelCatalan/MaterialSearchView#137]
I'm setting it as true like here
```java
mSearchView = (MaterialSearchView) findViewById(R.id.search_view);

mSearchView.setVoiceSearch(true);
        
mSearchView.showSearch(false);
```

Yet  the voice search icon in the searchbar is not displaying. Is this a bug? I'm on Marshmellow.

**Comment by agarwalmayank32** [comment]
@chrisjeon I agree with you. I have used this even on phones having lollipop but it is not showing the voice search icon. @MiguelCatalan please help us solve this.

**Comment by gnahtoad** [comment]
isVoiceAvailable() check is wrong, shouldn't it be:
...  return activities.size() > 0;

Right now it can't show the voice icon because of that check.

**Comment by ghost** [comment]
Thanks Miguel for your work on this. Is there any chance you can push a new version with this fix?
